tango/common/aliases.py

```diff
@@ -2,3 +2,11 @@
 from typing import Union
 
 PathOrStr = Union[str, PathLike]
+
+
+class EnvVarNames:
+    FILE_FRIENDLY_LOGGING_ENV_VAR = "FILE_FRIENDLY_LOGGING"
+    LOG_LEVEL_ENV_VAR: str = "TANGO_LOG_LEVEL"
+    CLICK_LOGGER_ENABLED_ENV_VAR: str = "TANGO_CLICK_LOGGER_ENABLED"
+    LOGGING_HOST_ENV_VAR: str = "TANGO_LOGGING_HOST"
+    LOGGING_PORT_ENV_VAR: str = "TANGO_LOGGING_PORT"
```

Comment:
I'm not super keen on this structure. Seems unnecessary complex, and then you have to use `__dict__` to take advantage of it. What's wrong with just a `Set[str]`?

I know in CS 101 they teach that you should never use raw literals, but in cases like this it buys you nothing. What's the advantage of always typing `MyClass.MY_LONG_NAME_VAR` instead of `"MY_LONG_NAME"`? I guess you get some resistance against typos. That's a pretty weak gain for abusing the class system like this.

Anyways, not a ship blocker or anything, but I don't like it.

Reply:
Yea, you're right. That's not great. I switched to using an Enum instead: fe943e5